staging/src/k8s.io/client-go/tools/watch/until.go

```diff
@@ -101,7 +101,9 @@ func UntilWithoutRetry(ctx context.Context, watcher watch.Interface, conditions
 // It guarantees you to see all events and in the order they happened.
 // Due to this guarantee there is no way it can deal with 'Resource version too old error'. It will fail in this case.
 // (See `UntilWithSync` if you'd prefer to recover from all the errors including RV too old by re-listing
-//  those items. In normal code you should care about being level driven so you'd not care about not seeing all the edges.)
+//
```

Comment:
lots of identantion changes, is that normal?

Reply:
Yeah, go1.19 gofmt changed dramatically :-(

Reply:
xref https://tip.golang.org/doc/go1.19#go-doc

Reply:
TIL

Reply:
Still an open issue against master to resolve incorrect changes gofmt made to comments - https://github.com/kubernetes/kubernetes/issues/111701